zetaclient/orchestrator/bootstrap.go

```diff
@@ -481,6 +478,20 @@ func syncObserverMap(
 	return added, removed, nil
 }
 
+func btcDatabaseFileName(chain chains.Chain) string {
```

Comment:
It seems a test can be trivial to add for this function

Reply:
https://github.com/zeta-chain/node/pull/3225/commits/54b452f77d31b8079aa32dc3d64793c970104329